<launch>

    <!-- Launch AR detection node -->
    <node pkg="rover" type="ar_detection_node.py" name="ar_detection_node" output="screen"/>

    <!-- Launch light beacon detection node -->
    <!-- <node pkg="rover" type="beacon_detection_blinking_color.py" name="light_beacon_detection_node" output="screen"/> -->

</launch>